### PR TITLE
remove pointless .gitignore files [master]

### PR DESCRIPTION
**Pull request by hannesa2**
Please make sure that the following boxes are checked before submitting your Pull Request, thank you!

- [x] You have signed the [Eclipse ECA](https://wiki.eclipse.org/ECA)
- [x] All of your commits have been signed-off with the correct email address (The same one that you used to sign the CLA)
- [ ] If This PR fixes an issue, that you reference the issue below. OR if this is a new issue that you are fixing straight away that you add some Description about the bug and how this will fix it.
- [ ] If this is new functionality, You have added the appropriate Unit tests.

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,4 +1,7 @@
-include ':org.eclipse.paho.android.service',':org.eclipse.paho.android.sample',':paho.mqtt.android.example'
+include ':org.eclipse.paho.android.service'
```
Comment (natelowry) [inline settings.gradle:1]: this change seems to have broken things on windows and on the travis CI build?

Comment (natelowry) [reply settings.gradle:1]: @icraggs @hannesa2 fyi that this line seems suspicious ^

thx for the library

Comment (hannesa2) [reply settings.gradle:1]: _"Things"_ .... what's broken here, when you split it in separate files ?
Please show us the Windows error.

CI builds properly https://travis-ci.org/github/eclipse/paho.mqtt.android/builds/671150922

image

Comment (hannesa2) [reply settings.gradle:1]: I tested on Linux:
image

I tested on macOS:
image

Then I tested previous/parent commit `git checkout b5d5e5ef91e846d39a365` to  
_"Move from minSDK 9 to min 14 (#245)"_

The result is the **same error** on both platforms.

### So it has nothing to do with this commit !

Btw, the `master` is in a very bad condition, last previous commit is 3 years old and gradle is `2.x`

Comment (hannesa2) [reply settings.gradle:1]: I recommend the maintainer do merge `develop` into `master` (and give up `develop`) to have one place of truth.
Btw, `develop` builds properly !

Comment (natelowry) [reply settings.gradle:1]: sounds good on `develop`

i was looking at the Travis CI logs.  the build started breaking once this was merged in:
https://travis-ci.org/github/eclipse/paho.mqtt.android/builds/694740637#L1585

although now that i see it the final error says gateway timeout, so maybe something external and just a coincidence?

```
A problem occurred configuring root project 'paho.mqtt.android'.
> Could not resolve all dependencies for configuration ':classpath'.
   > Could not download httpcore.jar (org.apache.httpcomponents:httpcore:4.1)
      > Could not get resource 'https://jcenter.bintray.com/org/apache/httpcomponents/httpcore/4.1/httpcore-4.1.jar'.
         > Could not GET 'https://jcenter.bintray.com/org/apache/httpcomponents/httpcore/4.1/httpcore-4.1.jar'. Received status code 504 from server: Gateway Time-out
```

thx for the info!

Comment (hannesa2) [reply settings.gradle:1]: In my point of view the master is dead. The new `master` is `develop`.
That's why I recommend to merge `develop` into `master` @icraggs

**Pull request merged**